modules/tappxBidAdapter.js

```diff
@@ -262,11 +262,24 @@ function buildOneRequest(validBidRequests, bidderRequest) {
   imp.secure = 1;
 
   imp.bidfloor = utils.deepAccess(validBidRequests, 'params.bidfloor');
+  if (utils.isFn(validBidRequests.getFloor)) {
+    try {
+      imp.bidfloor = validBidRequests.getFloor({
+        currency: CUR,
+        mediaType: '*',
+        size: '*'
+      }).floor;
```

Comment:
There is a scenario where somone has floors in a currency X

You ask for it in currency Y

But the currency conversion module is not loaded.

The getFloor function will return the floor in the currency X

And you will pass it along like so.

So, if you would like to be a bit more careful, most adapters check the responding currency is in the one they expected before sending it to their exchange endpoint.

Here is an example:

https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L1054-L1061

Reply:
Thanks!
We used the example for the change.
(We make the push of this change)